understanding/22/accessible-authentication-minimum.html

```diff
@@ -72,6 +72,7 @@ <h3>Cognitive Function Tests</h3>
             
             <p>Remembering a site-specific password is a <a>cognitive function test</a>. Such tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, or a pattern gesture to perform on a touch screen, cognitive function tests will exclude some people. When a cognitive function test is used, at least one other authentication method must be available which is not a cognitive function test.</p>
 
+            <p>Some <abbr title="Completely Automated Public Turing test to tell Computers and Humans Apart">CAPTCHA</abbr> systems have an audio alternative of the visible text that the user needs to transcribe. Unless the audio has been been transcribed, it can't be used as an alternative method.</p>            
```

Comment:
My concern with this phrasing is that I don't see how a provided transcription of the audio could then be any form of test. I think maybe it could be tweaked as follows:
```suggestion
            <p>Some <abbr title="Completely Automated Public Turing test to tell Computers and Humans Apart">CAPTCHA</abbr> systems have an audio alternative of the visible text. If the user needs to transcribe this audio, it can't be used as an accessible alternative method.</p>            
```